xbmc/FileItem.h

```diff
@@ -542,6 +547,7 @@ class CFileItem :
   PVR::CPVRRadioRDSInfoTagPtr m_pvrRadioRDSInfoTag;
   CPictureInfoTag* m_pictureInfoTag;
   std::shared_ptr<const ADDON::IAddon> m_addonInfo;
+  EventPtr m_eventLogEntry;
```

Comment:
I'm not a huge fan of this but since `CFileItem` is hopelessly overloaded anyway I'm not gonna object.